### ISSUE TITLE
Accessibility - iOS - DOL Seal Being Read as Button

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#99]
While testing the last few version of Sweat and Toil for iOS VoiceOver reads the DOL seal as "Button". It used to read something like"ILAB - Bureau of International Labor Affairs button" or  "United States Department of Labor Button".

By only saying "Button" a VoiceOver user has no idea what the use of the button is.

**Comment by jfc3-dol** [comment]
The DOL seal is still being read as a button only and not as "United States Department of Labor Button" or it should be made a link because it goes to ILAB's website "United States Department of Labor Link"

**Comment by jfc3-dol** [comment]
VoiceOver now reads the ILAB logo as "ILAB - Bureau of International Labor Affairs button" and the DOL seal as "United States Department of Labor button".

So I'm going to close the issue.
